pkgs/applications/misc/omorfi/hfst-python.nix

```diff
@@ -0,0 +1,28 @@
+{ icu
+, swig
+, pkgs
+}:
+
+let
+  hfst = pkgs.callPackage ./hfst.nix { };
+in
+  pkgs.python3Packages.buildPythonPackage {
+    inherit (hfst) src;
+    inherit (hfst) version;
+    inherit (hfst) meta;
+
+    pname = "hfst-python";
```

Comment:
Just in case: https://github.com/sikmir/nur-packages/blob/master/pkgs/linguistics/python-hfst/default.nix